src/server/debug/server/server.go

```diff
@@ -71,66 +71,67 @@ func (s *debugServer) handleRedirect(
 	redirect redirectFunc,
 	collect collectFunc,
 ) error {
+	ctx := pachClient.Ctx() // this context has authorization credentials we need
```

Comment:
It would be clearer to just add the context to `handleRedirect`’s signature, but perhaps that is better in a different PR?

Reply:
I had a variant of that, but it ends up being tricky... there are some calls made to pachd with the thick client, in which case it uses pachClient.Ctx(), and some calls made directly through the autogenerated grpc stubs, which need that exactly context (not server.Context(), which is missing grpc creds needed to make internal calls in the context metadata!)

What this means is that in the 3 callers of this method, they'd have to create a pachClient and put the context in, and then take the context out and pass it to this method.  Seems clearer to only pass the pachClient, so that it looks to the caller like we're doing something weird, which we are.